api/v1/controllers/lenses.js

```diff
@@ -265,10 +264,10 @@ module.exports = {
    */
   getLens(req, res, next) {
     const resultObj = { reqStartTime: req.timestamp };
-    const key = req.swagger.params.key.value;
+    const url = req.url;
 
     // try to get cached entry
-    redisCache.get(`${lensesRoute}/${key}`, (cacheErr, reply) => {
+    redisCache.get(url, (cacheErr, reply) => {
```

Comment:
This stuff must be really old to use callback! We should be using promises. But you can create a separate story for that ;)